pkg/analysis/passes/version/version_test.go

```diff
@@ -8,51 +8,17 @@ import (
 
 	"github.com/grafana/plugin-validator/pkg/analysis"
 	"github.com/grafana/plugin-validator/pkg/analysis/passes/metadata"
+	"github.com/grafana/plugin-validator/pkg/analysis/passes/published"
 	"github.com/grafana/plugin-validator/pkg/testpassinterceptor"
 	"github.com/jarcoal/httpmock"
 	"github.com/stretchr/testify/require"
 )
 
 const testPluginId = "test-plugin-panel"
 
-func getMockVersionResponse(id string, version string) string {
```

Comment:
I removed all the http mocking from this test and moved to the published one. now we mock the published analyzer response directly